environment.go

```diff
@@ -785,6 +793,13 @@ type BuildBaronProject struct {
 	BFSuggestionFeaturesURL string `mapstructure:"bf_suggestion_features_url" bson:"bf_suggestion_features_url"`
 }
 
+type AnnotationsSettings struct {
+	// a list of jira fields the user wants to display in addition to state assignee and priority
+	JiraCustomFields []string `mapstructure:"jira_custom_fields" bson:"jira_custom_fields"`
```

Comment:
I think we need a mapping of field names to display names. For instance, Jira could call it `custom_field_123`, so we'd need the user to tell us what to call it in the UI.

Alternatively, we could ask Jira what it's called when we search for the issue. It looks like there's an [extra parameter](https://developer.atlassian.com/cloud/jira/platform/rest/v3/intro/#expansion) (`expand=names`) that will tell us what all the fields are called.

Reply:
added a display text field